src/React.tasks.proj

```diff
@@ -52,7 +52,8 @@ of patent rights can be found in the PATENTS file in the same directory.
 					var contents = File.ReadAllText(path);
 					var newContents = versionRegex.Replace(
 						contents, 
-						string.Format("\"version\": \"{0}\"", Version)
+						string.Format("\"version\": \"{0}\"", Version),
+						1
```

Comment:
This is in order to just replace the first instance of version numbering in the .json files. This is done because now in RC2 there are a couple more places where you need to use version number, like when declaring CLI tools.